package.json

```diff
@@ -44,8 +63,7 @@
         "url": "https://github.com/freenowtech/wave.git"
     },
     "files": [
-        "lib/",
-        "codemods/"
```

Comment:
Why remove codemods?

Reply:
Not sure why it was done, I'll put them back